corehq/apps/sms/models.py

```diff
@@ -7,6 +7,8 @@
 from django.db import models
 from corehq.apps.users.models import CouchUser, CommCareUser
 from casexml.apps.case.models import CommCareCase
+from dimagi.utils.couch.migration import (SyncCouchToSQLMixin,
```

Comment:
Yeah Cory used this process to migrate all the XFormsSession docs to Postgres, and this just made a util for it